lte/gateway/c/session_manager/test/test_stored_state.cpp

```diff
@@ -279,6 +280,7 @@ TEST_F(StoredStateTest, test_stored_session) {
   EXPECT_EQ(stored_monitor.level, MonitoringLevel::PCC_RULE_LEVEL);
 
   EXPECT_EQ(stored.imsi, "IMSI1");
+  EXPECT_EQ(stored.pdp_start_time, 12345);
```

Comment:
Not sure if it is possible in other tests or if you will have access to this field, but can you add other checks like this in other of the tests you did.

Reply:
Yeah so the issue is in most cases it will be an actual unix timestamp so I can't just predict it